link_shortener/core/wait_for_db.py

```diff
@@ -1,8 +1,9 @@
+
 '''
 Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
 Licensed under the MIT (Expat) License (see LICENSE in Documentation).
 '''
 import time
 
 
-time.sleep(12)
+time.sleep(12)
```

Comment:
Tady ti pořád neukazuje to kolečko? (no newline at the end)

Reply:
Zbytek vypadá dobře. Když tam vrátíš tenhle řádek tak to rychle proklikám a nejspíš mergnu

Reply:
stale nemam kolecko :(
Done